tests/integration/test_apply_with_env_files.py

```
@@ -0,0 +1,220 @@
+"""
+Integration tests for haconiwa apply with --env flag
+"""
+import unittest
+from unittest.mock import patch, MagicMock
+import tempfile
+import shutil
+from pathlib import Path
+import yaml
```

Ruff (0.11.9)

`yaml` imported but unused

Remove unused import: `yaml`

(F401)